lib/rucio/core/replica.py

```diff
@@ -1083,12 +1084,15 @@ def delete_replicas(rse, files, ignore_availability=True, session=None):
     replica_rse = get_rse(rse=rse, session=session)
 
     if not (replica_rse.availability & 1) and not ignore_availability:
-        raise exception.ResourceTemporaryUnavailable('%s is temporary unavailable for deleting' % rse)
+        raise exception.ResourceTemporaryUnavailable('%s is temporary unavailable'
+                                                     'for deleting' % rse)
```

Comment:
This looks weird, I think an additional newline went in there?

Reply:
yes it looks weird. it's one way to break string on several lines.

Reply:
First time I am seeing this 😄

Reply:
it was not the most intuitive way for me too.  I got some inspiration from:
https://mail.python.org/pipermail/python-list/2013-November/661383.html